tests/testthat/test-2mergeSEs.R

```diff
@@ -331,7 +331,7 @@ test_that("mergeSEs", {
     expect_equal( rownames(tse), test )
     
     # Expect that each tip is found from rows when tree is pruned during merge.
-    test <- rowTree(tse)$tip.label
+    test <- sort(rowTree(tse)$tip.label)
     expect_equal( rownames(tse), test )
```

Comment:
This might not be the case

`rownames(tse) %in% rowTree(tse)$tip.label`

But this works only if the tips are names similarly to rows in original data.